web/crux-ui/src/routes.ts

```diff
@@ -105,3 +106,8 @@ export const deploymentDeployUrl = (productId: string, versionId: string, deploy
 export const teamsInviteUrl = (teamId: string) => `${ROUTE_TEAMS}/${teamId}/invite`
 export const teamsActiveUserApiUrl = (userId: string) => `${API_TEAMS_ACTIVE_USERS}/${userId}`
 export const teamAcceptInviteApiUrl = teamId => `${API_TEAMS}/${teamId}/accept`
+
+// notification
+export const notificationUrl = (id: string) => `${ROUTE_NOTIFICATIONS}/${id}`
+export const notificationApiUrl = (id?: string) => `/api${ROUTE_NOTIFICATIONS}` + (id ? `/${id}` : '')
```

Comment:
we need an API_NOTIFICATIONS constant
so we can get rid of the ternary in notificationApiUrl